docs/_edot-sdks/dotnet/setup/edot-defaults.md

```diff
@@ -257,7 +257,9 @@ OpenTelemetry SDK.
 | Option                   | EDOT .NET default | OpenTelemetry SDK default |
 | ------------------------ | ----------------- | ------------------------- |
 | IncludeFormattedMessage  | `true`            | `false`                   |
-| IncludeScopes            | `true`            | `false`                   |
+| IncludeScopes            | `true` (1.0.0 to 1.0.1) *           | `false`                   |
```

Comment:
Maybe it's more clear to reverse this if from now on it would be `false`?

Reply:
Yeah, good shout. I've updated.